scripts/04187-khansa.py

```diff
@@ -0,0 +1 @@
+print('Hello World, this is khansa AMROUNI with HNGi7 ID 04187 using Python for stage 2\n khansa.amrouni@gmail.com')
```

Comment:
Hi Khansa, please remove the `\n` and replace it with a ' task' as stated [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)

Reply:
thanks for your review! 
Done!